post_installation_configuration/connected-to-disconnected.adoc

```diff
@@ -8,17 +8,22 @@ toc::[]
 
 
 
-There might be some scenarios where you need to convert your {product-title} cluster from a connected cluster to a disconnected cluster.  
+There might be some scenarios where you need to convert your {product-title} cluster from a connected cluster to a disconnected cluster.
 
 A disconnected cluster, also known as a restricted cluster, does not have an active connection to the internet. As such, you must mirror the contents of your registries and installation media. You can create this mirror registry on a host that can access both the internet and your closed network, or copy images to a device that you can move across network boundaries.
 
-This topic describes the general process for converting an existing, connected cluster into a disconnected cluster.    
+This topic describes the general process for converting an existing, connected cluster into a disconnected cluster.
 
-:FeatureName: The process for converting a cluster from connected to disconnected 
+:FeatureName: The process for converting a cluster from connected to disconnected
 include::snippets/technology-preview.adoc[leveloffset=+1]
 
 include::modules/installation-about-mirror-registry.adoc[leveloffset=+1]
 
+[NOTE]
```

Comment:
Can you add this note actually into the above module (installation-about-mirror-registry.adoc) instead of in the assembly here? 

I ask because we recently added a new section on disconnected mirroring that uses the above module, and could also benefit from having this note in it.

The section I'm talking about is here: https://deploy-preview-42413--osdocs.netlify.app/openshift-enterprise/latest/installing/disconnected_install/installing-mirroring-disconnected.html

Reply:
Ack. Addressed